src/server/unix.rs

```diff
@@ -88,11 +82,14 @@ pub fn bootstrap(method: &dyn Method, settings: &init::Settings)
                     format!("error running server {:?}", cmd))?;
             init_credentials(&settings, &inst)?;
             drop(child);
-            println!("Bootstrap complete. To start a server:\n  \
-                      edgedb server start {}",
-                      settings.name.escape_default());
+            if settings.start_conf == StartConf::Manual {
+                println!("Bootstrap complete. To start a server:\n  \
```

Comment:
I'm not sure if it's `a server` or `the server` in this context. @elprans ?

Reply:
IDK, tutorials usually use `a server`. This is kinda the same and not the same :(

Reply:
Done